Extension/c_cpp_properties.schema.json

```diff
@@ -37,6 +37,8 @@
                             "c++11",
                             "c++14",
                             "c++17",
+                            "c++20",
+                            "c++Latest",
```

Comment:
We shouldn't use c++20 latest -- it should just be c++20.

Reply:
ok, fixed